content/content-tool/tool/src/webapp/vm/resources/sakai_dropbox_multiple_folders_upload.vm

```diff
@@ -229,4 +232,25 @@
 		#end
 
 	}
+	function handleFileChange()
+        {
+                const input = document.getElementById('MultipleFolderContent');
+                const feedback = document.getElementById('fileUploadFeedback');
+
+                if (input.files.length > 0)
+                {
+                        const fileName = input.files[0].name;
+		        const successTemplate = feedback.textContent;
+                        feedback.textContent = successTemplate.replace("{0}", fileName);
+                        feedback.style.display = 'block';
+
+                        setTimeout(() => {
+                        	feedback.style.display = 'none';
+                        }, 10000);
+                }
+                else
+                {
+                        feedback.style.display = 'block';
+		}
+	}
```

Comment:
What would happen if there are multiple files upload or its for single file upload only? I would use Template Literal and use bootstrap class instead of using inline styles.

Reply:
Thanks for your feedback, Kunal. This is for single file upload only, it doesn't allow me to upload multiple files. I changed the code to include template literals and bootstrap classes.

Reply:
We should set this to resolve.